docs/tcp/nginxk8slb.conf

```diff
@@ -1,22 +1,22 @@
-# NginxK8sLB Stream configuration, for L4 load balancing
+# NGINX Loadbalancer for K8s Solution - Stream configuration, for L4 load balancing
```

Comment:
I think we should use the full name "Kubernetes" here, e.g.: NGINX Loadbalancer for Kubernetes